src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -37,4 +40,14 @@ public Boolean isFree() {
     public Boolean isHighRisk() {
         return isHighRisk;
     }
+
+    /**
+     * @return A date string (e.g. 2017-04-24) to identify the date an email
+     *         address was first seen by MaxMind. This is expressed using the
+     *         ISO 8601 date format.
+     */
+    @JsonProperty("first_seen")
+    public String firstSeen() {
```

Comment:
I can change this.  Just wondering why the other methods in this class don't follow this same pattern?